### PR TITLE
Don't clobber keyboard target after executing action

### PR DESCRIPTION
**Pull request by pokey**
## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [-] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [-] I have not broken the cheatsheet

### CODE REVIEW

File: packages/cursorless-vscode/src/keyboard/KeyboardCommandsTargeted.ts [hunk 1]
```diff
@@ -261,6 +261,18 @@ export default class KeyboardCommandsTargeted {
       // For some Cursorless actions, it is more convenient if we automatically
       // exit modal mode
       await this.modal.modeOff();
+    } else {
+      // If we're not exiting cursorless mode, preserve the keyboard mark
+      // FIXME: Better to just not clobber the keyboard mark on each action?
```
Comment (josharian) [inline packages/cursorless-vscode/src/keyboard/KeyboardCommandsTargeted.ts:266]: i think it is funny that we outlawed TODO so now just use FIXME.

**Review by josharian — APPROVED** [review]
thanks! i just hit this myself, appreciate the fix

File: packages/cursorless-vscode/src/keyboard/KeyboardCommandsTargeted.ts (hunk 1 above)
Comment (pokey) [reply packages/cursorless-vscode/src/keyboard/KeyboardCommandsTargeted.ts:266]: TODO is for things that must be fixed before merge; FIXME is for things we don't like but are ok to merge. I guess we should prob add a note about that to contributor docs

Fwiw we've always tried to use this convention, but now it's enforced so those TODOs can actually give peace of mind

File: packages/cursorless-vscode-e2e/src/suite/keyboard/basic.vscode.test.ts [hunk 2]
```diff
@@ -69,6 +69,13 @@ const testCases: TestCase[] = [
     keySequence: ["da", "aw", "wp"],
     finalContent: "(a)",
   },
+  {
+    name: "reset keyboard target",
```
Comment (pokey) [inline packages/cursorless-vscode-e2e/src/suite/keyboard/basic.vscode.test.ts:73]:
```suggestion
    name: "preserve keyboard target",
```

**Pull request merged**